.gitignore

```diff
@@ -23,3 +23,6 @@ test/**/out
 
 # Editors
 **/.idea
+
+# example output
+examples/**/out
```

Comment:
Shouldn't this be specific to the example? 🤔

Reply:
I can make this specific to the examples in particular, but it seemed like almost all the examples I was working with had the output folder as `out`, but that's admittedly a fairly dangerous assumption. 

There doesn't seem to be any `.gitignore` files being used in any examples currently, should we start using them in order to use example specific gitignores?